fairworkflows/rdf_wrapper.py

```diff
@@ -1,7 +1,10 @@
 import warnings
 import rdflib
 
-from nanopub import Nanopub
+# TODO: use this:
```

Comment:
TODO!

Reply:
Ok, I only now see this comment. It was because of [this](https://github.com/fair-workflows/nanopub/pull/12/files), but is fixed now.